Client/mods/deathmatch/logic/luadefs/CLuaVehicleDefs.cpp

```diff
@@ -156,6 +157,7 @@ void CLuaVehicleDefs::LoadFunctions()
         {"setVehicleWheelScale", ArgumentParser<SetVehicleWheelScale>},
         {"setVehicleModelWheelSize", ArgumentParser<SetVehicleModelWheelSize>},
         {"spawnVehicleFlyingComponent", ArgumentParser<SpawnVehicleFlyingComponent>},
+        {"setPlaneTrailEnabled", ArgumentParser<SetSmokeTrailEnabled>},
```

Comment:
Why not call the functions set/is SmokeTrailEnabled like the c++ function names, instead of "plane trail"?

Reply:
or is/setVehicleSmokeTrailEnabled

Reply:
to make way simple

Reply:
Might still be good to keep it generic. You never know if someday someone makes it possible to have smoke trails behind any vehicle (ex. custom vehicles).

Reply:
okay I will rename